homeassistant/components/hassio/const.py

```diff
@@ -51,12 +63,17 @@
 ATTR_URL = "url"
 ATTR_REPOSITORY = "repository"
 
-
 DATA_KEY_ADDONS = "addons"
 DATA_KEY_OS = "os"
 DATA_KEY_SUPERVISOR = "supervisor"
 DATA_KEY_CORE = "core"
 
+ISSUE_ID_UNHEALTHY = "unhealthy_system"
```

Comment:
If the constant is only used in a single module I'd define the constant in that module to make it easier to read the module.